### PR TITLE
Center aligned the search icon

### PR DESCRIPTION
**Pull request by Jigar3**
Screenshot of the change
edited
#16

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
         <!--Import Google Icon Font-->
         <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
         <!--Import materialize.css-->
-        <!-- <link type="text/css" rel="stylesheet" href="css/materialize.css"/>-->
```
Comment (udiboy1209) [inline index.html]: Why is this needed?

Comment (Jigar3) [reply index.html]: My mistake
